src/demos.py

```diff
@@ -211,16 +211,15 @@ def pipeline_demo():
         print("Invalid Choice")
 
     try:
-        choice = str(
-            input(
-                f"Use the Regionalatlas? (y/N)\n"
-            )
-        )
+        choice = str(input(f"Use the Regionalatlas? (y/N)\n"))
         if choice == "y" or choice == "Y":
-            steps.append(RegionalAtlas(force_refresh=True))
+            choice = str(
+                input(f"Do you want to force execution if the data is present? (y/N)\n")
+            )
+            force_execution = choice == "y" or choice == "Y"
+            steps.append(RegionalAtlas(force_refresh=force_execution))
     except ValueError:
-        print("Invalid Choice")    
-
+        print("Invalid Choice")
```

Comment:
Maybe you can use the new `get_yes_no_input()` function

Reply:
I will take a look into it